docs/2.0/docs/pipelines/installation/addingexistingrepo.md

```diff
@@ -1,346 +1,430 @@
-* NOTE: As of writing this in October 2024 the above template points to a [boilerplate](https://github.com/gruntwork-io/boilerplate) template; the file is meant to be rendered by boilerplate into a usable file.  Please reach out to your Gruntwork solutions architect for assistance if you need help rendering the template.
+    ```yaml title="accounts.yml"
+    # required: Name of an account
+    $$AWS_ACCOUNT_NAME$$:
+      # required: The AWS account ID
+      id: "$$AWS_ACCOUNT_ID$$"
+      # required: The email address of the account owner
+      email: "$$AWS_ACCOUNT_EMAIL$$"
+    ```
 
-## Basic Configuration
+### Pipelines Configurations
 
-The minimum configurations required for Pipelines to operate correctly will vary depending on context. For the most common usage, Pipelines will need to be able to determine how to authenticate with a cloud provider in order to run Terragrunt commands. If it is not able to do so in a context where it is required, Pipelines will throw an error.
+Create a file called `.gruntwork/gruntwork.hcl` in the root of your repository with the contents below. This file will be used to configure Pipelines for your repository. Update the following values:
 
-The following is an example of a minimal configuration for a single Terragrunt unit that tells Pipelines how to authenticate with AWS using OIDC:
+- <CustomizableValue id="ENVIRONMENT_NAME" /> to a name that represents the environment you are deploying to. e.g. `production`, `staging`, `development`, etc.
+- <CustomizableValue id="PATH_TO_ENVIRONMENT" /> to the root-relative path of the folder in your repository that contains the terragrunt units for the environment you are deploying to. This may be the same as the environment name if there is a directory in the root of the repository that contains all the terragrunt units for the environment.
+- <CustomizableValue id="AWS_ACCOUNT_ID" /> to the AWS Account ID where terragrunt units matching the environment will be deployed to.
+- <CustomizableValue id="DEPLOY_BRANCH_NAME" /> to the branch name that you would like to deploy from. This is the branch that will trigger the Pipelines apply workflow when changes are merged to it. e.g `main`, `master`, etc. All other pull requests against this branch will trigger the Pipelines plan workflow.
 
-```hcl
-# gruntwork.hcl
-unit {
-  authentication {
-    aws_oidc {
-      account_id         = "an-aws-account-id"
-      plan_iam_role_arn  = "arn:aws:iam::an-aws-account-id:role-to-assume-for-plans"
-      apply_iam_role_arn = "arn:aws:iam::an-aws-account-id:role-to-assume-for-applies"
-    }
-  }
+```hcl title=".gruntwork/gruntwork.hcl"
+# Configurations applicable to the entire repository https://docs.gruntwork.io/2.0/docs/pipelines/installation/addingexistingrepo#repository-blocks
```

Comment:
Why are we documenting things as code comments here? Can't we just explain this in markdown above the code fence?

Reply:
+1

Reply:
I think that's useful when templating, but links are nicer to read in markdown.